_config.yml

```diff
@@ -2,8 +2,7 @@ title: "ESLint - Pluggable JavaScript linter"
 repository: "eslint/eslint.github.io"
 exclude: [
   'node_modules',
-  'bower_components',
-  'vendor'
+  'src'
```

Comment:
Since it's not ignoring assets/build, this ensures Jekyll waits to regenerate the site until after Webpack finishes and outputs the bundles (I don't think this really matters what order they run in, but this prevents Jekyll from regenerating the site when `src` files are edited and then once more when the bundles are outputted).